scheduler/src/cook/mesos/util.clj

```diff
@@ -132,6 +132,11 @@
   [db job-ent]
   (some-> job-ent :job/container remove-datomic-namespacing))
 
+(defn job-ent->group-uuid
+  "Take a job entity and return its group UUID"
+  [job-ent]
+  (some-> job-ent :group/_job first :group/uuid))
```

Comment:
Can a job be in more than 1 group? Asking because of the `first`

Reply:
I already spoke to @pschorf about it since the groups are returned as a set. A job can only belong to one group.